_source/logzio_collections/_log-sources/platform-service/zipkin.md

```diff
@@ -0,0 +1,102 @@
+* **Limit** must be 1000 traces or less
+
+## To integrate Zipkin server and Logz.io
+
+{: .tasklist .firstline-headline}
+1. Download Zipkin server and Zipkin-Logz.io Trace Storage
+
+    Download [Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec).
+
+    ```shell
+    wget -O zipkin.jar 'https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec'
+    ```
+
+    Download the [Zipkin-Logz.io Trace Storage](https://github.com/logzio/zipkin-logzio/releases) jar to the same directory.
+
+2. Run Zipkin server with the Logz.io extension
+
+    You can configure the Logz.io extension with shell variables or environment variables.
+
+    For a complete list of options, see the parameters below the code block.👇
+
+    ```bash
+    STORAGE_TYPE=logzio \
+    LOGZIO_ACCOUNT_TOKEN=<ACCOUNT-TOKEN> \
+    LOGZIO_LISTENER_HOST=<LISTENER-URL> \
+    LOGZIO_API_TOKEN=<API-TOKEN> \
+    LOGZIO_API_HOST=<API-URL> \
+    java -Dloader.path='zipkin-logzio.jar,zipkin-logzio.jar!lib' -Dspring.profiles.active=logzio -cp zipkin.jar org.springframework.boot.loader.PropertiesLauncher
+    ```
+
+    <div class="info-box tip">
+      You can optionally run two Zipkin-Logzio Trace Storage instances if you want to separate shipping and reading of your traces.
+
+      If you do, then the required fields change a bit from what's shown in the Parameters list:
+
+      * The **shipping instance** requires `STORAGE_TYPE=logzio`, `LOGZIO_ACCOUNT_TOKEN`, and `LOGZIO_LISTENER_HOST` (if you're not shipping to the default listener).
+      * The **reading instance** requires `STORAGE_TYPE=logzio`, `LOGZIO_API_TOKEN`, and `LOGZIO_API_HOST`.
+    </div>
+
+    {: .inline-header }
+    Parameters
+
+    STORAGE_TYPE=logzio <span class="required-param"></span>
+    : We wish there was a way to include this as a default.
+      Alas, Zipkin needs it, so you'll need to include this bit.
+
+    LOGZIO_ACCOUNT_TOKEN <span class="required-param"></span>
+    : Required when using as a collector to ship logs to Logz.io. \\
+      {% include log-shipping/replace-vars.html token=true %}
+      <!-- logzio-inject: account-token -->
+
+    LOGZIO_API_TOKEN <span class="required-param"></span>
+    : Required to read back traces from Logz.io. \\
+      Replace `<API-TOKEN>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use.
+
+    LOGZIO_LISTENER_HOST <span class="default-param">`listener.logz.io`</span>
+    : {% include log-shipping/replace-vars.html listener=true %}
+      <!-- logzio-inject: listener-url -->
+
+    LOGZIO_API_HOST <span class="default-param">`api.logz.io`</span>
+    : Required to read back spans from Logz.io. \\
+      Replace `<API-URL>` with your region's base API URL.
+      For more information on finding your account's region, see [Account region]({{site.baseurl}}/user-guide/accounts/account-region.html).
+
```

Comment:
@imnotashrimp please add the 2 new parameters

Reply:
In the latest commit, should show up soon